microsoft-automerge.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+      "github>gsoft-inc/renovate-config"
+  ],
+  "packageRules": [
+    {
+      "groupName": "microsoft",
+      "matchPackageNames": [
```

Comment:
Should we have the same pattern that was existing in default: 

```
      "matchPackageNames": [
        "dotnet-sdk",
        "dotnet-sdk",
        "mcr.microsoft.com/dotnet/sdk",
        "mcr.microsoft.com/dotnet/sdk",
        "mcr.microsoft.com/dotnet/aspnet",
        "mcr.microsoft.com/dotnet/aspnet",
        "mcr.microsoft.com/dotnet/runtime",
        "mcr.microsoft.com/dotnet/runtime",
        "mcr.microsoft.com/dotnet/runtime-deps",
        "mcr.microsoft.com/dotnet/runtime-deps",
        "/^Azure\\.Identity$/i",
        "/^Azure\\.Identity$/i",
        "/^Microsoft\\./i",
        "/^Microsoft\\./i",
        "/^System\\./i",
        "/^System\\./i",
        "/^coverlet\\.collector$/i",
        "/^coverlet\\.collector$/i",
        "/^xunit(\\.|$)/i",
        "/^xunit(\\.|$)/i",
        "/^MongoDB\\./i"
        "/^MongoDB\\./i"
      ],
```

Reply:
Do we consider mongo, xunit and coverlet packages to be Microsoft packages?

Reply:
I don't think so, maybe we can have another "trusted-thirdparties-automerge" or something similar?

BTW, xunit have created build issues recently because they add new Roslyn analyzers in patch release. Because we use WarningsAsErrors it can break the build.

Reply:
Yeah sounds good to create a seperate one for third party dependencies. Should the name be `dotnet-trusted-thirdparty-dependencies.json`, I'm thinking at some point there could also be one for frontend or other

Reply:
If you create multiple json, you can also add one that extends all of them.